example/angular_material/conf.js

```diff
@@ -0,0 +1,27 @@
+// An example configuration file.
+exports.config = {
+    directConnect: true,
+  
+    // Capabilities to be passed to the webdriver instance.
+    capabilities: {
+      'browserName': 'chrome'
+    },
+  
+    // Framework to use. Jasmine is recommended.
+    framework: 'jasmine',
+  
+    // Spec patterns are relative to the current working directory when
+    // protractor is called.
+    specs: [
+        'input_spec.js',
+        'mat_paginator_spec.js'
+    ],
+	
+	// Disable promise manager because we are going to use async/await
+	SELENIUM_PROMISE_MANAGER: false,
+  
+    // Options to be passed to Jasmine.
+    jasmineNodeOpts: {
+      defaultTimeoutInterval: 30000
+    }
+};
```

Comment:
End Of File (`EOF`). Or it is not necessary?

Reply:
Hi, @CrispusDH ! Thanks for the review) Actually, I tried to follow code style of [examples](https://github.com/angular/protractor/blob/master/example/example_spec.js). But I can add it if necessary.